### PR TITLE
IGVF-1682-add-restricted-bucket

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: config/pyramid/ini/development.ini [hunk 1]
```diff
@@ -34,8 +34,8 @@ in_docker = true
 cors_trusted_origins =
     http://localhost:3000
 file_upload_bucket = igvf-files-local
+restricted_file_upload_bucket = igvf-restricted_files-local
```
Comment (keenangraham) [inline config/pyramid/ini/development.ini:37]: Need to fix `igvf-restricted_files-local` -> `igvf-restricted-files-local`

File: config/pyramid/ini/loader.ini [hunk 2]
```diff
@@ -34,8 +34,8 @@ in_docker = true
 cors_trusted_origins =
     http://localhost:3000
 file_upload_bucket = igvf-files-local
+restricted_file_upload_bucket = igvf-restricted_files-local
```
Comment (keenangraham) [inline config/pyramid/ini/loader.ini:37]: Need to fix `igvf-restricted_files-local` -> `igvf-restricted-files-local`

**Review by keenangraham — CHANGES_REQUESTED** [review]

File: config/pyramid/ini/development.ini (hunk 1 above)
Comment (gabdank) [reply config/pyramid/ini/development.ini:37]: done

File: config/pyramid/ini/loader.ini (hunk 2 above)
Comment (gabdank) [reply config/pyramid/ini/loader.ini:37]: done

**Review by keenangraham — DISMISSED** [review]

**Pull request merged**